imagebuildah/executor.go

```diff
@@ -300,6 +302,7 @@ func (b *Executor) startStage(ctx context.Context, stage *imagebuilder.Stage, st
 		volumeCacheInfo: make(map[string]os.FileInfo),
 		output:          output,
 		stage:           stage,
+		logger:          b.logger,
```

Comment:
This field isn't needed, since it's never different from the Executor's logger.  Use the StageExecutor's `executor.logger` in those places instead, similar to how we handle several other fields that we refer to in the Executor.